.github/workflows/dep_trig_release.yml

```diff
@@ -36,24 +36,35 @@ jobs:
       env:
         PR_URL: ${{ github.event.pull_request.html_url }}
         GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Wait for status checks
+      uses: WyriHaximus/github-action-wait-for-status@v1.7.1
+      with:
+        ignoreActions: "Merge a dependabot PR"
+        checkInterval: 10
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Why is this required?

Reply:
`gh pr merge --auto --squash "$PR_URL"` will not block Action, while the PR merge will wait until status check ends. Next step `push tag` will continue immediately after this command.

It we don't wait before push tag, then the tag will be pushed at previous(HEAD^1) commit.

Reply:
So the current step is:

1. dependabot upgrade envd
2. trigger CI lint/test/build and auto merge + push tag
3. new tag trigger release (by action `on.push.tags`) but push tag also trigger release (by `uses release.yml`)

Reply:
> So the current step is:
> 
> 1. dependabot upgrade envd
> 2. trigger CI lint/test/build and auto merge + push tag
> 3. new tag trigger release (by action `on.push.tags`) but push tag also trigger release (by `uses release.yml`)

`new tag trigger release (by action `on.push.tags`)` only happens when user push a tag, if the tag is pushed by Action, it will not be triggered, so I turned to last one to straight triger it in previous commit.

If we don't need to tag and release some special version of `envd-lsp` not according to `envd` manually, the first one could be removed. @kemingy